lib/cdo/honeybadger.rb

```diff
@@ -26,6 +26,7 @@ def self.notify_command_error(command, status, stdout, stderr)
         backtrace: backtrace,
         context: {
             stdout: stdout,
+            stderr: stderr,
```

Comment:
Is the error_message = stderr a few lines above still necessary (useful)?

Isn't stderr already logged in backtrace?